psse/psse-model/src/main/java/com/powsybl/psse/model/io/LegacyTextReader.java

```diff
@@ -147,7 +147,8 @@ private static String processText(String line) {
                 result.append(part);
             } else {
                 // Outside quotes: process the txt
-                result.append(part.replaceAll("\\s+,", ",")
+                // On the next line: "(?<=\S|^)" = backtracking protection. The previous char should be a non-white char or a line start.
+                result.append(part.replaceAll("(?<=\\S|^)\\s+,", ",") // see comment above
```

Comment:
The `|^` might not be necessary, to be checked
```suggestion
                result.append(part.replaceAll("(?<=\\S)\\s+,", ",") // see comment above
```

Reply:
I don't think it is really needed because leading white spaces are trimmed from `part` in the current state of the code, but it's maybe safer to keep it.